core/dbt/include/global_project/docs/overview.md

```diff
@@ -26,7 +26,7 @@ On model pages, you'll see the immediate parents and children of the model you'r
 button at the top-right of this lineage pane, you'll be able to see all of the models that are used to build,
 or are built from, the model you're exploring.
 
-Once expanded, you'll be able to use the `--models` and `--exclude` model selection syntax to filter the
+Once expanded, you'll be able to use the `--select` and `--exclude` model selection syntax to filter the
```

Comment:
Following the change we made recently in #3791. This is already called `--select` within the dbt-docs site itself, and has been for some time.